scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -261,7 +256,7 @@
    "source": [
     "# save output as CSV\n",
     "combined_normalized_data.to_csv(os.path.join(single_cell_dir, 'normalized_data.csv'), index=False)\n",
-    "combined_transformed_data.to_csv(os.path.join(single_cell_dir, 'transformed_data.csv'), index=False)\n"
+    "combined_transformed_data.to_csv(os.path.join(single_cell_dir, 'transformed_data.csv'), index=False)"
```

Comment:
Not particularly relevant for this review, should we make the output name configurable in an earlier cell (like where file paths are asked for)?  I ask because of issue #120.